handbook/business-operations/people-operations.md

```diff
@@ -242,6 +242,8 @@ There are some defaults that we use for these agreements:
 
 Then hit send!  After all of the signatures are there, the completed document will automatically be uploaded to the appropriate Google Drive folder, and a Slack message will appear in the `#help-classified` channel.
 
+Once the agreement is signed and in the appropriate folder, an [issue](https://fleetdm.com/handbook/business-operations#intake) is created to provide access to Slack and any relevant company tools.
```

Comment:
This makes sense!

To connect the dots, part of this is also adding the consultant to KPIs so the manager notification ritual can happen and we can track hours worked.  (I can see how you're planning to have that live in the issue template which makes sense!)

https://github.com/fleetdm/fleet/pull/13376/files#r1297967301

More of a consistency thing: since this is a step that has to be manually taken (rather than a description of an automated process like above), we should express it in the imperative mood (i.e. "To hire a US-based consultant, send them an agreement.")  This makes it clear that action needs to be taken by the person following these steps.

@JoStableford fyi I'll just edit this inline so this can get merged prior to Sam's migration